universite-du-quebec-a-montreal.csl

```diff
@@ -1,487 +1,542 @@
         <multiple>trad.</multiple>
       </term>
-      <term name="translator" form="verb-short">
-        <single>trad. par</single>
-        <multiple>trad. par</multiple>
+      <term name="no date" form="short">s.d.</term>
+      <term name="in">Dans</term>
+      <term name="retrieved">Récupéré de</term>
+      <term name="presented at">Communication présentée à</term>
+      <term name="page" form="short">
+        <single>p.</single>
+        <multiple>p.</multiple>
       </term>
-      <term name="presented at">Communication</term>
     </terms>
   </locale>
-  <macro name="secondary-contributors">
-    <choose>
-      <if type="chapter paper-conference" match="none">
-        <group delimiter=". ">
-          <choose>
-            <if variable="author">
-              <names variable="editor">
-                <label form="verb-short" text-case="capitalize-first" suffix=". "/>
-                <name and="text" delimiter=", "/>
-              </names>
-            </if>
-          </choose>
-          <choose>
-            <if variable="author editor" match="any">
-              <names variable="translator">
-                <label form="verb-short" text-case="capitalize-first" suffix=" "/>
-                <name and="text" delimiter=", "/>
-              </names>
-            </if>
-          </choose>
-        </group>
-      </if>
-    </choose>
-  </macro>
   <macro name="container-contributors">
     <choose>
-      <if type="chapter paper-conference" match="any">
-        <group prefix=", " delimiter=", ">
-          <choose>
-            <if variable="author">
-              <names variable="editor">
-                <label form="verb-short" prefix=" " text-case="lowercase" suffix=" "/>
-                <name and="text" delimiter=", "/>
-              </names>
-              <choose>
-                <if variable="container-author">
-                  <group>
-                    <names variable="container-author">
-                      <label form="verb-short" prefix=" " text-case="lowercase" suffix=" "/>
-                      <name and="text" delimiter=", "/>
-                    </names>
-                  </group>
-                </if>
-              </choose>
-            </if>
-          </choose>
-          <choose>
-            <if variable="author editor" match="any">
-              <names variable="translator">
-                <label form="verb-short" prefix=" " text-case="lowercase" suffix=" "/>
-                <name and="text" delimiter=", "/>
-              </names>
-            </if>
-          </choose>
-        </group>
+      <if type="chapter paper-conference entry-encyclopedia entry-dictionary" match="any">
+        <names variable="editor container-author" delimiter=", " suffix=", ">
+          <name and="text" initialize-with=". " delimiter=", "/>
+          <label form="short" prefix=" (" text-case="title" suffix=")"/>
+          <label form="short" prefix=" (" suffix=")" text-case="lowercase"/>
```

Comment:
Nevermind. I copied that schema fragment from "master", where we already fixed this post-1.0.1: https://github.com/citation-style-language/schema/commit/850a86fb56f10c198ef051159df07386d19dd56a